src/ray/gcs/redis_module/ray_redis_module.cc

```diff
@@ -406,18 +402,21 @@ Status TableEntryToFlatbuf(RedisModuleKey *table_key, RedisModuleString *entry_i
                                        fbb.CreateVector(&data, 1));
     fbb.Finish(message);
   } break;
-  case REDISMODULE_KEYTYPE_ZSET: {
+  case REDISMODULE_KEYTYPE_LIST: {
+    RedisModule_CloseKey(table_key);
```

Comment:
Why call `CloseKey`? We have automatic Redis GC, so we shouldn't need to ever call this I think.

Is there some weird interaction since this key gets opened again by `RedisModule_Call`? If so, please document.